test/Ocelot.UnitTests/Kubernetes/KubeServiceDiscoveryProviderTests.cs

```diff
@@ -38,7 +38,7 @@ public KubeServiceDiscoveryProviderTests()
         {
             _serviceName = "test";
             _namespaces = "dev";
-            _port = 8001;
+            _port = 86;
```

Comment:
Was this an intentional change? Or this a machine specific setting somehow?

Reply:
Yes. Intentional. Please see the PR description that has the reasoning.